packages/block-library/src/button/save.js

```diff
@@ -33,17 +33,21 @@ export default function save( { attributes } ) {
 	// A title will no longer be assigned for new or updated button block links.
 
 	return (
-		<div>
-			<RichText.Content
-				tagName="a"
-				className={ buttonClasses }
-				href={ url }
-				title={ title }
-				style={ buttonStyle }
-				value={ text }
-				target={ linkTarget }
-				rel={ rel }
-			/>
-		</div>
+		<>
+			{ text && (
```

Comment:
This could be simplified to just `return text && (...`. No need for a Fragment and `{`.

Reply:
Actually, I just had a better idea. Use an early return instead, for example:

```js
if ( ! text ) {
	return null;
}
```

Reply:
I just realized that the Fragment being the root element may actually be the cause of the test failures. The `wp-block-button` and `aligncenter` classes are not being applied to the `div` anymore.